app/assets/stylesheets/components/_header.scss

```
@@ -1,8 +1,43 @@
-.govuk-header__logo,
-.govuk-header__content {
-  width: unset;
-}
+.app-header {
+  @include govuk-media-query($from: tablet) {
+    .govuk-header__container {
+      display: flex;
+      justify-content: space-between;
+
+      .govuk-header__logo,
+      .govuk-header__content {
+        width: unset;
+      }
+
+      .govuk-header__logo {
+        flex-grow: 1;
+      }
+    }
+  }
+
+  &.app-header--full-border {
+    border-bottom-color: govuk-colour("dark-grey");
+
+    .govuk-header__container {
+      margin-bottom: 0;
+      border-bottom: none;
+    }
+
+    &.app-header--production {
+      border-bottom-color: govuk-colour("blue");
+    }
+
+    &.app-header--qa {
+      border-bottom-color: govuk-colour("orange");
+    }
+
+    &.app-header--staging {
+      border-bottom-color: govuk-colour("red");
+    }
```

Would it make sense to name these classes according to their colours, and then use the existing `env => colour` mapping defined in `PhaseBannerHelper`?

```suggestion
    &.app-header--blue {
      border-bottom-color: govuk-colour("blue");
    }

    &.app-header--orange {
      border-bottom-color: govuk-colour("orange");
    }

    &.app-header--red {
      border-bottom-color: govuk-colour("red");
    }
```

That way we'll have one single source of truth for environment colours:

https://github.com/DFE-Digital/itt-mentor-services/blob/279f3db10905c878e7b6810acfc835c4b0064f29/app/helpers/phase_banner_helper.rb#L2-L8

And we can use the new `phase_banner_color` helper method in the view:

```ruby
<%= govuk_header(homepage_url: "/", classes: "app-header app-header--#{phase_banner_color} app-header--full-border") do |header| %>
```